plugins/inventory/scaleway.py

```diff
@@ -31,6 +31,9 @@
         tags:
             description: Filter results on a specific tag.
             type: list
+        scw_profile:
+            description: The config profile to use in config file
+            type: string
```

Comment:
```suggestion
            type: string
            version_added: 4.4.0
```

Reply:
Thanks for review, i added it in last commit